### PR TITLE
Add InputDateView

### PR DESCRIPTION
**Pull request by lukoktonos**
Add an InputDateView which is basically a copy of InputDateTimeView without the Time elements.

Used with new QDM statusDate on careGoal (need to use new models, there are `qdm-5.5-debug-only` branches of parsers, execution, converter you can point to to load the new MAT package and see a careGoal with this new view in Bonnie.

Pull requests into Bonnie require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.
- [ ] Update cqm-execution ref after https://github.com/projecttacoma/cqm-execution/pull/61

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] This PR is into the correct branch.
- [x] JIRA ticket for this PR: https://jira.mitre.org/browse/BONNIE-2076
- [x] JIRA ticket links to this PR
- [x] Code diff has been done and been reviewed (it **does not** contain: additional white space, not applicable code changes, debug statements, etc.)
- [x] If UI changes have been made, google WAVE plug-in has been executed to ensure no 508 issues were introduced.
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass (remember to update Gemfile when applicable)
- [x] Test fixtures updated and documented as necessary ( see [internal wiki](https://gitlab.mitre.org/bonnie/internal-documentation/wikis/testing#test-fixtures) )
- [x] Code coverage has not gone down and all code touched or added is covered. 
- [x] Automated regression test(s) pass N/A

**Reviewer 1:**

Name: @hossenlopp 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

**Reviewer 2:**

Name:
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

### CODE REVIEW

File: app/assets/javascripts/views/patient_builder/data_criteria_attribute_editor.js.coffee [hunk 1]
```diff
@@ -138,7 +139,7 @@ class Thorax.Views.DataCriteriaAttributeEditorView extends Thorax.Views.BonnieVi
     # If this is an any type, there will be more options than one.
     else if info.instance == 'Any'
       # TODO: Filter these more if possible
-      return ['Code', 'Quantity', 'DateTime', 'Ratio', 'Integer', 'Decimal', 'Time']
+      return ['Date', 'Code', 'Quantity', 'DateTime', 'Ratio', 'Integer', 'Decimal', 'Time']
```
Comment (hossenlopp) [inline app/assets/javascripts/views/patient_builder/data_criteria_attribute_editor.js.coffee]:
```suggestion
      return ['Code', 'Quantity', 'Ratio', 'Integer', 'Decimal', 'Date', 'DateTime', 'Time']
```

Comment (lukoktonos) [inline app/assets/javascripts/views/patient_builder/data_criteria_attribute_editor.js.coffee]: Remove 'Date' from this dropdown ('Date') is only used for statusDate

File: app/assets/javascripts/views/patient_builder/data_criteria_attribute_editor.js.coffee [hunk 2]
```diff
@@ -138,7 +139,7 @@ class Thorax.Views.DataCriteriaAttributeEditorView extends Thorax.Views.BonnieVi
     # If this is an any type, there will be more options than one.
     else if info.instance == 'Any'
       # TODO: Filter these more if possible
-      return ['Code', 'Quantity', 'DateTime', 'Ratio', 'Integer', 'Decimal', 'Time']
+      return ['Code', 'Quantity', 'Ratio', 'Integer', 'Decimal', 'DateTime', 'Time']
```
Comment (hossenlopp) [inline app/assets/javascripts/views/patient_builder/data_criteria_attribute_editor.js.coffee]: `Date` should be in this list.

**Review by hossenlopp — CHANGES_REQUESTED** [review]
It works! But...

**Review by hossenlopp — APPROVED** [review]
Approve assuming Gemfile, package.json and lockfiles will be updated.

**Review by jbradl11 — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -108,6 +108,6 @@
   "scripts": {},
   "dependencies": {
     "browserify": "^16.2.3",
-    "cqm-execution": "https://github.com/projecttacoma/cqm-execution/#2019-standards-update"
```
Comment (hossenlopp) [inline package.json:111]: This branch specification needs to stay.

**Review by hossenlopp — CHANGES_REQUESTED** [review]

**Review by hossenlopp — APPROVED** [review]
